tsconfig.json

```diff
@@ -18,6 +18,7 @@
     ],
     "pretty": true,
     "resolveJsonModule": true,
+    "strict": true,
```

Comment:
This is good, but you didn't change the CI script, so actually it's not verified during pipeline. This `tsconfig.json` is a bit stale anyway. You can check [deno_std](https://github.com/denoland/deno) for example how to run CI with strict mode.

Reply:
rad